src/WebSocketsClient.h

```diff
@@ -80,6 +80,9 @@ class WebSocketsClient: private WebSockets {
 
         void setAuthorization(const char * user, const char * password);
         void setAuthorization(const char * auth);
+	
+        void setExtraHeaders(const char * extraHeaders = NULL);
+        void setExtraHeaders(char * extraHeaders);

```

Comment:
where is the Implementation of `void setExtraHeaders(char * extraHeaders);`? 
may simple remove it. `const char *` will work fine for text data.